<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetMotorPar</class>
 <widget class="QDialog" name="SetMotorPar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>电机参数设置</string>
   </property>
   <widget class="QLineEdit" name="lE_Horizon_Set">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>146</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>61</y>
      <width>98</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>步进电机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>旋转方向</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>水平旋转周期</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>178</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cB_Mo_1">
       <property name="text">
        <string>Motor1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cB_Mo_2">
       <property name="text">
        <string>Motor2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>144</y>
      <width>166</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="Dir_1">
       <item>
        <property name="text">
         <string>顺时针</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>逆时针</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Dir_2">
       <item>
        <property name="text">
         <string>顺时针</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>逆时针</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>198</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pB_Confirm">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_Exit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
